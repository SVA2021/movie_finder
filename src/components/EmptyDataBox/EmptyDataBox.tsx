import {Typography} from '@mui/material';
import React, {memo} from 'react';

export const EmptyDataBox = memo(() => {
  return (
    <Typography
      variant="h3"
      component={'p'}
      fontStyle={'italic'}
      textAlign={'center'}
      minHeight={{xs: '100px', lg: '200px'}}
      lineHeight={{xs: '100px', lg: '200px'}}
      color={'action.hover'}
    >
      Нет данных
    </Typography>
  );
});