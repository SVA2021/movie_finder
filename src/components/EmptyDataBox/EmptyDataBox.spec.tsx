import {render, screen, } from "@testing-library/react";
import {EmptyDataBox} from "./EmptyDataBox";

describe('render EmptyDataBox', () => {

  const setUp = () => render(<EmptyDataBox />);

  it('snapshot test', () => {
    setUp();
    expect(screen.getByTestId('empty-data-box')).toMatchSnapshot();
  });

  it('check text', () => {
    setUp();
    expect(screen.getByText(/нет данных/i)).toBeInTheDocument();
  });

})