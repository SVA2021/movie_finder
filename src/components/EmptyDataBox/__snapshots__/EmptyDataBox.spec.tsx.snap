// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render EmptyDataBox snapshot test 1`] = `
<p
  class="MuiTypography-root MuiTypography-h3 css-13mkr37-MuiTypography-root"
  data-testid="empty-data-box"
>
  Нет данных
</p>
`;
